pkg/api/vlabs/validate.go

```diff
@@ -124,7 +125,12 @@ func (o *OrchestratorProfile) Validate(isUpdate bool) error {
 						return fmt.Errorf("enableDataEncryptionAtRest is only available in Kubernetes version %s or greater; unable to validate for Kubernetes version %s",
 							minVersion, o.OrchestratorVersion)
 					}
-
+					if o.KubernetesConfig.EtcdEncryptionKey != "" {
+						_, err = base64.StdEncoding.DecodeString(o.KubernetesConfig.EtcdEncryptionKey)
+						if err != nil {
+							return fmt.Errorf("etcdEncryptionKey must be base64 encoded. Please provide a valid base64 encoded value or leave the etcdEncryptionKey empty to auto-generate the value")
```

Comment:
@ritazh I added base64 encoding validation here as follow up to your comment in #2694, let me know if you know of better way of checking if a string is base64 encoded...